scss/window/_layout.scss

```diff
@@ -53,6 +53,11 @@
         flex-shrink: 0;
         align-items: center;
         vertical-align: top;
+
+        .k-button {
+            margin-top: calc(#{-1*$button-padding-y} - 1px);
+            margin-bottom: calc(#{-1*$button-padding-y} - 1px);
```

Comment:
`add-two($button-padding-y, 1px, -1)` might be cleaner here. Is there a variable that has the same meaning as the `1px`, for example the button border width, or the titlebar border width?

Please drop me a note (or assign me for review) when the styles affect an Angular component.